### ISSUE TITLE
About validation set and test set

### ISSUE THREAD

**Issue by Lnan1615** [Wenxuan-1119/TransBTS#36]
I divided my data into training sets, validation sets, and test sets. When training, I use "mode=train", but when I learn the code, I find that the validation set is not in training, and there is no parameter adjustment. In test.py, although there is an option to opt out of validation at training time, the setup of a single dataset file does not satisfy both validation and testing.
我将我的数据分为训练集、验证集和测试集。训练时，使用"mode=train"，但我学习代码时发现验证集并没有进入训练，没有进行参数调整.在test.py中，尽管有选择在训练时未进行验证的选项，但只有一个数据集文件的设置， 并不能同时满足验证和测试。
